projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -247,6 +248,29 @@ public static function style( $style ) {
 	public static function style_on() {
 		return self::style( true );
 	}
+	/**
+	 * Adds a quick link to the admin bar for the contact form entries.
+	 *
+	 * @param \WP_Admin_Bar $admin_bar The admin bar object.
+	 */
+	public static function add_quick_link_to_admin_bar( \WP_Admin_Bar $admin_bar ) {
+
+		if ( ! current_user_can( 'edit_pages' ) ) {
+			return;
+		}
+
+		$url = ( new Dashboard_View_Switch() )->get_forms_admin_url();
+
+		$admin_bar->add_menu(
+			array(
+				'id'     => 'jetpack-forms',
+				'parent' => null,
+				'group'  => null,
+				'title'  => '<span class="dashicons dashicons-feedback" style="font: normal 20px/1 dashicons; display: block; float: left; margin-top: 6px; color: rgba(240, 246, 252, 0.6); margin-right: 6px;"></span><span class="ab-label">' . esc_html__( 'Form Responses', 'jetpack-forms' ) . '</span>',
```

Comment:
```suggestion
				'title'  => '<span class="dashicons dashicons-feedback ab-icon" style="top: 2px;"></span><span class="ab-label">' . esc_html__( 'Form Responses', 'jetpack-forms' ) . '</span>',
```

The number of inline styles needed here strikes me as odd.

We can get rid of them all if we re-use the `ab-icon` class on the dashicons `span`, like the other admin bar items.